test/utils/math/Math.t.sol

```diff
@@ -201,14 +201,31 @@ contract MathTest is Test {
         vm.assume(xyHi >= d);
 
         // we are outside the scope of {testMulDiv}, we expect muldiv to revert
-        try this.muldiv(x, y, d) returns (uint256) {
-            fail();
-        } catch {}
+        vm.expectRevert();
+        Math.mulDiv(x, y, d);
     }
 
-    // External call
-    function muldiv(uint256 x, uint256 y, uint256 d) external pure returns (uint256) {
-        return Math.mulDiv(x, y, d);
+    // MOD EXP
+    function testModExp(uint256 b, uint256 e, uint256 m) public {
+        if (m == 0) {
+            vm.expectRevert(); // panic 0x12
```

Comment:
I think we can do:

```suggestion
            vm.expectRevert(abi.encode(0x12));
```

Reply:
Hmm, seems like Foundry's `expectRevert` doesn't catch panics.

```shell
[FAIL. Reason: Error != expected error:  != ; counterexample: calldata=0x8a59e100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000 args=[0, 0, 0]] testModExp(uint256,uint256,uint256) (runs: 17, μ: 16029, ~: 3918)
Traces:
  [3721] MathTest::testModExp(0, 0, 0)
    ├─ [0] VM::expectRevert()
    │   └─ ← ()
    └─ ← panic: division or modulo by zero (0x12)
```

Reply:
It doesn but you must encode them properly.

You can do 
`abi.encodeWithSignature("Panic(uint256)", 0x12)` or use the stdErrors from foundry.

Reply:
Look at you! Such an experience Foundry user 🥂